karma.conf.js

```diff
@@ -156,10 +156,6 @@ module.exports = function (config) {
         // how many browser should be started simultaneous
         concurrency: Infinity,
 
-        junitReporter: {
-            outputDir: 'karma-reports',
```

Comment:
Can we also remove this from `.gitignore`?

Reply:
ah, good shout